pkg/azure/disk.py

```diff
@@ -90,34 +91,33 @@ async def start_cloning(project, hostname):
         public_ip = Discover.objects(project=project,host=hostname).allow_filtering()[0]['public_ip']
         user = Project.objects(name=project).allow_filtering()[0]['username']
         storage = Storage.objects(project=project).allow_filtering()[0]['storage']
-        accesskey = Storage.objects(project=project).allow_filtering()[0]['access_key']
+        access_key = Storage.objects(project=project).allow_filtering()[0]['access_key']
         container = Storage.objects(project=project).allow_filtering()[0]['container']
-        sas_token = sas.generate_sas_token(storage,accesskey)
+        sas_token = sas.generate_sas_token(storage,access_key)
         url = "https://" + storage + ".blob.core.windows.net/" + container + "/"
         load_dotenv()
+        username = Project.objects(name=project)[0]['username']
         mongodb = os.getenv('BASE_URL')
         current_dir = os.getcwd()
-        os.popen('echo null > ./logs/ansible/migration_log.txt')
-        if hostname == "all":
-            command = "ANSIBLE_HOST_KEY_CHECKING=False /usr/local/bin/ansible-playbook -i "+current_dir+"/ansible/"+project+"/hosts "+current_dir+"/ansible/azure/start_migration.yaml -e \"url="+url+" sas='"+sas_token+"' mongodb="+mongodb+ " project="+project+" hostname="+hostname+"\""
-        else:
-            command = "ANSIBLE_HOST_KEY_CHECKING=False /usr/local/bin/ansible-playbook -i "+current_dir+"/ansible/"+project+"/hosts "+current_dir+"/ansible/azure/start_migration.yaml -e \"url="+url+" sas='"+sas_token+"' mongodb="+mongodb+ " project="+project+" hostname="+hostname+"\" --limit "+public_ip+" --user "+user+" --become-user "+user+" --become-method sudo"
-            print(command)
-            logger(command,"warning")
-        process = await asyncio.create_subprocess_shell(command, stdin = PIPE, stdout = PIPE, stderr = STDOUT)
-        await process.wait()
-        machines = BluePrint.objects(project=project).allow_filtering()
-        machine_count = len(machines)
-        flag = True
-        status_count = 0
-        while flag:
-            for machine in machines:
-                if int(machine['status'])>=25:
-                    status_count = status_count + 1
-            if status_count == machine_count:
-                flag = False
-        con.shutdown()
-        return not flag
+        playbook = "start_migration.yaml"
+        stage = "start clone"
```

Comment:
Use start_clone. But cloning might be better as the log file would appear as cloning _log.txt then